src/jwplayer/utils/videoTracking.ts

```diff
@@ -79,7 +78,7 @@ export const PROPERTY_NAMES = {
 };
 
 function addRunTimeParams(trackingParams: TrackData): TrackData {
-	const trackDataObject: TrackData = { ...addGlobalProps(), ...trackingParams };
```

Comment:
I don't know why these global props were added here but they are basically duplicated from tracking lib so I removed it from here.

Reply:
I appreciate the clarification. However, for future reference, it might be advisable to include a `@todo` comment instead and avoid altering the code within unrelated PR since, as you mentioned, the reason behind adding these global properties is unknown 😉

Reply:
Actually, it's a bit related because these global props overwrite user id as well but instead of removing just this one I removed all of them. :) 

Also, I like to follow "The Boy Scouting Rule" (https://levelup.gitconnected.com/the-boy-scouting-rule-a-simple-yet-effective-programming-principle-5d504c7dd993) so I always try to clean up instead of putting todo comments if possible

Reply:
Thanks for clarification! It helps more than the original PR when it was added ;) https://github.com/Wikia/jwplayer-fandom/pull/85

Reply:
I'm aware of the rule, but due to its P1 status, I strongly recommend we exercise additional caution in the future 😄